benches/run_agentic_benchmark.py

```
@@ -0,0 +1,1123 @@
+        return (self.path / rel).read_text(encoding="utf-8")
+
+    def exists(self, rel: str) -> bool:
+        return (self.path / rel).exists()
+
+    def list_files(self) -> list[str]:
+        return sorted(p.name for p in self.path.iterdir() if p.is_file())
+
+    def run_terminal(self, command: list[str], timeout: int = 30) -> dict[str, Any]:
+        start = time.time()
+        try:
+            proc = subprocess.run(command, cwd=self.path, capture_output=True,
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)